zonec.c

```diff
@@ -1132,6 +1136,7 @@ zparser_conv_svcbparam_key_value(region_type *region,
 		return zparser_conv_svcbparam_ech_value(region, val);
 	case SVCB_KEY_ALPN:
 		return zparser_conv_svcbparam_alpn_value(region, val, val_len);
+	case SVCB_KEY_DOHPATH:
```

Comment:
If there is output of escapes for non printable characters, here is where it then needs to be unescaped, I guess.

Reply:
This isn't used in the other "simple" SVCB params either

Reply:
An unescaped value is already passed to `zparser_conv_svcbparam_key_value()` so it doesn't need to be unsecaped here again (except with alpn values which are double-escaped).